WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.m

```diff
@@ -661,6 +672,49 @@ - (void)showUpdateSiteIconAlert
     [self presentViewController:updateIconAlertController animated:YES completion:nil];
 }
 
+- (void)uploadDroppedSiteIconImage:(UIImage *)image
+{
+    self.imageCropViewController = [[ImageCropViewController alloc] initWithImage:image];
+    self.imageCropViewController.maskShape = ImageCropOverlayMaskShapeSquare;
+   
+    __weak __typeof(self) weakSelf = self;
+    self.imageCropViewController.onCompletion = ^(UIImage *image, BOOL modified) {
+        [weakSelf dismissViewControllerAnimated:YES completion:nil];
+        if (!modified) {
```

Comment:
Do we want to allow users to use their image without any modifications? It might already be the correct size / shape, so they might not want to change the cropping.